api/SVGGeometryElement.json

```diff
@@ -247,90 +423,122 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGGeometryElement/pathLength",
           "support": {
-            "chrome": {
-              "version_added": "1"
-            },
-            "chrome_android": {
-              "version_added": "18"
-            },
-            "edge": {
-              "version_added": "≤79"
-            },
-            "firefox": {
-              "version_added": "53"
-            },
-            "firefox_android": {
-              "version_added": "53"
-            },
-            "ie": {
-              "version_added": false
-            },
-            "opera": {
-              "version_added": "≤12.1"
-            },
-            "opera_android": {
-              "version_added": "≤12.1"
-            },
-            "safari": {
-              "version_added": "3"
-            },
-            "safari_ios": {
-              "version_added": "1"
-            },
-            "samsunginternet_android": {
-              "version_added": "1.0"
-            },
-            "webview_android": {
-              "version_added": "1"
-            }
-          },
-          "status": {
-            "experimental": false,
-            "standard_track": true,
-            "deprecated": false
-          }
-        }
-      },
-      "supportOtherThanPath": {
-        "__compat": {
-          "description": "Support for elements other than <code>&lt;path&gt;</code>",
-          "support": {
-            "chrome": {
-              "version_added": null
-            },
-            "chrome_android": {
-              "version_added": null
-            },
+            "chrome": [
+              {
+                "version_added": "57"
+              },
+              {
+                "version_added": "1",
+                "version_removed": "57",
+                "partial_implementation": true,
+                "notes": "Only supported for <a href='https://developer.mozilla.org/docs/Web/API/SVGPathElement'><code>SVGPathElement</code></a>, not all <code>SVGGeometryElement</code> objects."
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "57"
+              },
+              {
+                "version_added": "18",
+                "version_removed": "57",
+                "partial_implementation": true,
+                "notes": "Only supported for <a href='https://developer.mozilla.org/docs/Web/API/SVGPathElement'><code>SVGPathElement</code></a>, not all <code>SVGGeometryElement</code> objects."
+              }
+            ],
             "edge": {
-              "version_added": null
-            },
-            "firefox": {
-              "version_added": "61"
-            },
-            "firefox_android": {
-              "version_added": "61"
-            },
+              "version_added": "79"
```

Comment:
This is not a mistake. `pathLength` wasn't supported in IE or EdgeHTML, confirmed by testing.